back/config/initializers/1_locales.rb

```diff
@@ -1,9 +1,9 @@
 require 'i18n/backend/fallbacks'
 
-CL2_SUPPORTED_LOCALES = %i[en ar-SA en-GB en-CA nl-BE nl-NL fr-BE fr-FR de-DE it-IT
+CL2_SUPPORTED_LOCALES = %i[en ar-MA ar-SA en-GB en-CA nl-BE nl-NL fr-BE fr-FR de-DE it-IT
                            da-DK nb-NO es-ES es-CL pl-PL hu-HU kl-GL lb-LU ro-RO pt-BR mi sr-Latn sr-SP hr-HR].freeze
 
-I18n.available_locales = CL2_SUPPORTED_LOCALES + %i[nl fr de it da nb es pl hu kl lb ro pt ar sr hr]
+I18n.available_locales = CL2_SUPPORTED_LOCALES + %i[nl fr de it da nb es pl hu kl lb ro pt ar sr hr ma]
```

Comment:
2 letter code doubts:

Cannot use `ar` (I presume) as already in use, so I have used `ma` here (which matches with my choice of filename for `back/config/blocked_words/ma.txt`.

This would be the only time we use the 2nd pair of letters from the locale code for the 2 letter code, so I am very unsure about this, atm.

I subsequently settled on using `ar-ma` where typically we would use `ar` in the FE locales (after trying `ma`, so this use of `ma` in the BE is inconsistent with the FE, and I am also unsure if this might cause issues.

Reply:
@lilkraftwerk Thanks, I also forgot to ask about the filnemae I have used for `back/config/blocked_words/ma.txt`. If we don't need `ma` here, then this filename would be the only place we use `ma`. I'm not sure this makes sense.

Reply:
Plan: delete file as we will default to using `ar.txt`